scripts/_fm-creator.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+const postFolder = '../_posts';
+
+// Frontmatter Creator
+fs.readdir(postFolder, (err, files) => {
+  files.forEach(f => {
+    if (f === path.basename(__filename)) return;
+
+    fs.readFile(path.join(postFolder, f), 'utf-8', (_, data) => {
+      // New frontmatter data
+      const frontmatter = {
+        title: 'Title goes here',
+        id: 'Id goes here',
+        description: 'Description goes here',
+        url: path.join('./_posts', f),
```

Comment:
We have to rework on this: If I were to use this as is the `url` would become something like: `https://blog-gen.now.sh/./_posts/[filename]`.

I think just leaving it with a default text is enough:

```suggestion
        url: 'URL goes here',
```